examples/with-storybook-styled-jsx-scss/package.json

```diff
@@ -13,8 +13,7 @@
     "next": "^10.0.0",
     "sass": "^1.32.4",
     "react": "17.0.1",
-    "react-dom": "17.0.1",
-    "styled-jsx-plugin-sass": "^1.0.0"
```

Comment:
In fact, we still need to have `@styled-jsx/plugin-sass` in `devDependencies` and `sass` should in `devDependencies`.

Reply:
Actually, we don't need `@styled-jsx/plugin-sass` because `styled-jsx` is shipped with `NextJS` and it works out of the box.

And yeah, you are right. I will put `sass` in dev dependencies now.

Reply:
Yes `styled-jsx` is there out of the box in `next` but it is not the case for this plugin, to use it, you need to install it on your own.

Reply:
Actually, you are right. My bad. I added it now.